### DISCUSSION TITLE
[BUG] [SP-API] [Amazon SP-API Self authorization]

### DISCUSSION THREAD

**Discussion by MaritzaK1** [amzn/selling-partner-api-models#2805] [General]
I'm new to amazon sp api. I think I did the steps correctly. I am attaching all the data I have as a picture.
We've been talking like this for days and they just send me videos and documents. There is something I missed. Please help me. Thank you very much.
Sorry for having too many pictures. But I don't know how to explain anymore.
Note: We will only use the app for our own store. Draft shouldn't be a problem.
As you can see below, I can get access tokens and refresh tokens.
But when I try to connect as below it does not accept. I am sending x-amz-access-token.

![image](https://user-images.githubusercontent.com/63651151/233189906-8e465e51-a261-446e-82c7-8544fd3eb028.png)
![image](https://user-images.githubusercontent.com/63651151/233189932-611884e5-bda9-4d07-a459-f137ef4ed990.png)
![image](https://user-images.githubusercontent.com/63651151/233189952-f9c54ec9-344d-4e6a-a190-72498786f1a1.png)
![image](https://user-images.githubusercontent.com/63651151/233189986-33c5d15f-50a4-47ba-85e1-0b35fec62ad8.png)
![image](https://user-images.githubusercontent.com/63651151/233190094-69926b70-8c90-4154-bc60-0ddb7befb98a.png)
![image](https://user-images.githubusercontent.com/63651151/233190138-11b23afe-df47-44c0-bfe5-426b025a54dd.png)
![image](https://user-images.githubusercontent.com/63651151/233190158-76a6c945-ba75-4838-bdcd-b4857625d550.png)
![image](https://user-images.githubusercontent.com/63651151/233190182-d540a96f-e83d-4aaf-b31f-93c3ac8e6976.png)
![image](https://user-images.githubusercontent.com/63651151/233190199-6bae3dc5-6853-443a-97a4-c9181ce43e6f.png)

**Comment by MaritzaK1** [comment]
Please help.

**Comment by MaritzaK1** [comment]
@amazon-auto

**Comment by sungolivia** [comment]
Hey @MaritzaK1 can you show me where it tells you to go on LWA console? I see that you have a screenshot with LWA console open..

**Comment by sarbadaj** [comment]
@MaritzaK1 Did you find any solution for this?

**Comment by MaritzaK1** [comment]
@sarbadaj Yes i found
These are missing in the header.
![image](https://github.com/amzn/selling-partner-api-docs/assets/63651151/baa77c65-ddc0-4053-bb47-3c4cb2723452)

**Reply by MaritzaK1** [reply]
And 
![image](https://github.com/amzn/selling-partner-api-docs/assets/63651151/fef29843-ba6c-46ad-8310-cc4894c27448)
this.

**Reply by sarbadaj** [reply]
I have tried same as above but still facing the same issue

**Comment by Geek2017** [comment]
try downloading the postman model that required your used case here:
https://developer-docs.amazon.com/sp-api/docs/sp-api-models
